bootstrapvz/providers/ec2/tasks/ami.py

```diff
@@ -125,3 +125,12 @@ def run(cls, info):
             registration_params['sriov_net_support'] = 'simple'
 
         info._ec2['image'] = info._ec2['connection'].register_image(**registration_params)
+
+        # Setting up tags on the AMI
+        if 'tags' in info.manifest.data and len(info.manifest.data['tags']) > 0:
+            def apply_format(v):
```

Comment:
This lambda seems unnecessary, all you are doing is currying one argument, better to inline it to improve readability. Great idea to use the manifest vars btw. it's very applicable in this case.